demo-notebooks/additional-demos/remote_ray_job_client.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Submit a job using an authorized Ray dashboard and the Job Submission Client remotely\n",
```

Comment:
Submit a Job remotely to Ray Dashboard behind oAuth.
```suggestion
    "# Submit a training job remotely to Ray Dashboard protected by oAuth.\n",
```

I'm thinking something more along this line, when the user is looking for the example, they don't need to know how, aka using the job submission client, they would want to search their use case so i'm thinking this is a bit clearer. wdyt?